src/Raven.Studio/typescript/models/database/index/indexDefinition.ts

```diff
@@ -353,6 +353,11 @@ class indexDefinition {
        this.additionalAssemblies.remove(assemblyItem);
     }
     
+    setMapsAndReduce(mapsFromHistory: string[], reduce: string) {
```

Comment:
what if maps from not from history. Will it work ? ;-) I assume you can pass any map here.

Reply:
done. 
`mapsFromHistory => maps`